resources/assets/components/InboxItem/InboxTile.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class InboxTile extends React.Component {
+  displayImage(photo_url) {
```

Comment:
@katiecrane yea sorry that was confusing, i think it's just a semantic thing. Like it could be `getImageUrlFromProp(photoProp)` because it's weird to seem like you could pass a `photo_url` that could be `default`.